src/frontend-proxy.ts

```diff
@@ -0,0 +1,105 @@
+import { getSubdomain, getPathname } from './url-utils'
+
+export const FRONTEND_DOMAIN = 'frontend-sooty-ten.now.sh'
+const API_ENDPOINT = 'https://api.serlo.org/graphql'
+const FRONTEND_ALLOWED_TYPES = [
+  'Article',
+  'Applet',
+  'Course',
+  'CoursePage',
+  'Page',
+  'TaxonomyTerm',
+  'Video',
+]
+const FRONTEND_PROBABILITY = 0.1
+
+export async function handleRequest(
+  request: Request,
+  probability = FRONTEND_PROBABILITY,
+  allowdTypes = FRONTEND_ALLOWED_TYPES
+): Promise<Response | null> {
+  const url = request.url
+  const path = getPathname(url)
+
+  if (getSubdomain(url) !== 'de') return null
+
+  if (path === '/enable-frontend')
+    return createResponse('Enable frontend', probability, true)
+  if (path === '/disable-frontend')
+    return createResponse('Disable frontend', probability, false)
+
+  if (path.startsWith('/_next/') || path.startsWith('/_assets/'))
+    return await fetchBackend(true, false)
+
+  if (path !== '/') {
+    const typename = await queryTypename(path)
+    if (typename === null || !allowdTypes.includes(typename)) return null
+  }
```

Comment:
@Entkenntnis With `landing-page` you meant that a request to `/` (main page) should also be able to be forwarded to frontend or did you meant other landing pages as well?

Reply:
Jup, I meant /